plugins/permission-node/src/integration/createPermissionIntegrationRouter.ts

```diff
@@ -174,75 +177,98 @@ type NoInfer<T> = T extends infer S ? S : never;
 export const createPermissionIntegrationRouter = <
   TResourceType extends string,
   TResource,
->(options: {
-  resourceType: TResourceType;
-  // Do not infer value of TResourceType from supplied rules.
-  // instead only consider the resourceType parameter, and
-  // consider any rules whose resource type does not match
-  // to be an error.
-  rules: PermissionRule<TResource, any, NoInfer<TResourceType>>[];
-  getResources: (
-    resourceRefs: string[],
-  ) => Promise<Array<TResource | undefined>>;
-}): express.Router => {
-  const { resourceType, rules, getResources } = options;
+>(
+  permissionApi: PermissionAuthorizer,
+  permissions: Permission[],
```

Comment:
> I think having a `resourceType` argument can be especially confusing here - the jenkins permission has a resource type, so the plugin author, if using this helper in its current state, may be tempted to include the catalog entity resource type as an argument. I think there's room for a combination of parameter renaming, docs, or maybe even a restructuring of the helper entirely in some way. I briefly considered splitting the new well-known endpoint into an entirely separate helper, but I veered away from that as I thought having a single router integration point is much less for the plugin author to keep track of.

In the current version, I moved the `permissions: Permission[]` argument out of the options object and placed it as a top level argument to the function. I think this helps semantically separate the list of permissions from all the stuff that's related to conditional filters. That said, I think we can take it one step further and rename `options` to something else, like `conditionOptions`. Thought on the name?